scripts-dev/release.py

```diff
@@ -89,10 +92,12 @@ def prepare() -> None:
     """
 
     # Make sure we're in a git repo.
-    repo = get_repo_and_check_clean_checkout()
+    synapse_repo = get_repo_and_check_clean_checkout()
+    sytest_repo = get_repo_and_check_clean_checkout("../sytest", "sytest")
 
     click.secho("Updating git repo...")
```

Comment:
As someone who needs to touch his YubiKey for each push/pull, it'd be good to be explicit about two `pull` operations here.

```suggestion
    click.secho("Updating Synapse and Sytest git repos...")
```

Reply:
I had no idea that was a thing, but sure :-)